Rubberduck.Resources/RubberduckUI.de.resx

```diff
@@ -1532,13 +1532,18 @@ Import abgebrochen.</value>
 
 Import abgebrochen.</value>
   </data>
-  <data name="UpdateFromFilesCommand_DocumentDoesNotExist" xml:space="preserve">
+  <data name="ImportCommand_DocumentDoesNotExist" xml:space="preserve">
     <value>Es existiert kein Dokumentenmodul namens '{0}', s.d. die Datei '{1}' nicht importiert werden kann.
 
 Import abgebrochen.</value>
   </data>
-  <data name="UpdateFromFilesCommand_UserFormDoesNotExist" xml:space="preserve">
-    <value>Es existiert keine Userform namens '{0}', s.d. kein Codebehind einer Userform aus der Datei '{1}' überschrieben werden kann.
+  <data name="ImportCommand_BinaryAndComponentDoNotExist" xml:space="preserve">
+    <value>Für die Datei '{0}' fehlen die benötigte(n) Binärdatei(en) '{1}' und die Zielkomponente '{2}' existiert noch nicht.
+
+Import abgebrochen.</value>
+  </data>
+  <data name="ImportCommand_BinaryDoesNotExist" xml:space="preserve">
+    <value>Für die Datei '{0}' fehlen die benötigte(n) Binärdatei(en) '{1}'.
 
 Import abgebrochen.</value>
```

Comment:
Ought the resource contain line breaks?  (Note: my comment applies to other resource keys with line breaks; I will not repeat it for those)

Reply:
Yes, it is supposed to contain line breaks. They are necessary to get one blank line between the reason for the failure and the statement that the import is aborted in the message box.